docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc

```diff
@@ -99,8 +115,6 @@ NOTE: Users are valid for both passport and {Keycloak} auth providers.
 |`123`
 |===
 
-For a complete list of available users, see the link:https://github.com/feedhenry-raincatcher/raincatcher-server/blob/master/src/modules/passport-auth/users.json[gitub repository]
+For all available users, see: https://github.com/feedhenry-raincatcher/raincatcher-server/blob/master/src/modules/passport-auth/users.json[users.json]
 
-NOTE: The Mobile and Portal Applications contain sample Steps, Workflows and Workorders.
```

Comment:
This was removed from original content.

Reply:
I moved this to the top of the section. It seems kind of strange to me as a note at the bottom. Putting this at the top gave a bit more context to the purpose for running the apps locally. It also is more descriptive than saying 'demo environment'.

Can add this back in as a next step or something such as:

"Now that the Portal and Mobile applications are running on your local machine, you can log in to them and evaluate the sample Steps, Workflows, and Workorders."